sdk/python/tests/test_historical_retrieval.py

```diff
@@ -335,7 +335,9 @@ def test_historical_features_from_bigquery_sources(
     ) = generate_entities(start_date, infer_event_timestamp_col)
 
     # bigquery_dataset = "test_hist_retrieval_static"
-    bigquery_dataset = f"test_hist_retrieval_{int(time.time())}"
+    bigquery_dataset = (
+        f"test_hist_retrieval_{int(time.time_ns())}_{random.randint(0, 1000)}"
```

Comment:
One more minor nit, can we maybe use `randint(1000, 9999)` so that we have a fixed width dataset name?

Reply:
Sure